### PR TITLE
feat: Add Footer

### PR DESCRIPTION
**Pull request by ishita-1305**

### CODE REVIEW

**Comment by ashutosh-rath02** [comment]
image
- There is alignment issues.
- Improve the project structure as suggested earlier
@ishita-1305

File: src/app/Homepage.jsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
 import React from 'react'
 
-import { Navbar } from './components/Navbar'
+import { Navbar } from './components/marginals/navbar2'
```
Comment (Srish-ty) [inline src/app/Homepage.jsx]: change to 
import { Navbar } from './components/marginals/Navbar'

File: src/app/signup/page.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ import {
   signUpWithGitHub,
   signUpWithGoogle
 } from '../../firebase/signupAuth'
-import { Navbar } from '../components/Navbar'
+import { Navbar } from '../components/marginals/navbar2'
```
Comment (Srish-ty) [inline src/app/signup/page.jsx]: make changes
import { Navbar } from './components/marginals/Navbar'

**Review by Srish-ty — CHANGES_REQUESTED** [review]
also delete navbar2.jsx 
and yarn.lock files

**Pull request merged**